frontend/src/app/academics/academics-admin/room/room-editor/room-editor.component.html

```diff
@@ -65,6 +65,15 @@
           name="capacity"
           required />
       </mat-form-field>
+      <mat-form-field appearance="outline" color="accent">
+        <mat-label>Room Description</mat-label>
+        <input
+          matInput
+          placeholder=""
+          formControlName="description"
+          name="description"
+          required />
```

Comment:
Thoughts on removing `required` from this field? - since description is a new field (and all rooms do not have a description by default), going to edit any room will now force the user to add a description if they want to save. Could always default to an empty string.

Reply:
Looping in @aarjavjain2002